releases.md

```diff
@@ -16,39 +16,30 @@ _Updated April 26, 2023_
-| 1.3.8         | January 26th     | February 2nd   |
-| 2.6.0         | February 21th    | February 28th  |
-| 1.3.9         | March 9th        | March 16th     |
-| 2.7.0         | April 17th       | May 2nd        |
-| 1.3.10        | May 11th         | May 18th       |
-| 2.8.0         | May 30th         | June 6th       |
-| 1.3.11        | June 22nd        | June 29th      |
-| 2.9.0         | July 11th        | July 24th      |
-| 1.3.12        | August 3rd       | August 10th    |
-| 2.10.0        | September 05th   | TBD            |
+### 2023 ###
+| Release Number| First RC Generated (release window opens) | Latest Possible Release Date (release window closes) | Release Date |
+|:--------------|:-----------------|:---------------|:---------------|
+| 2.10.0        | September 05th   | September 19th |
 | 1.3.13        | September 14th   | September 21st |
-| 2.11.0        | October 10th     | TBD       |
+| 2.11.0        | October 10th     | October 24th   |
 | 1.3.14        | November 9th     | November 16th  |
-| 2.12.0        | November 30th    | TBD      |
+| 2.12.0        | November 30th    | December 14th  |
 
 OpenSearch [follows semver](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/), which means we will only release breaking changes in major versions.  All minor versions are compatible with every other minor version for that major.  For example, 1.2.0 will work with 1.3.2, 1.4.1, etc, but may not work with 2.0.
  
-For minor version releases, OpenSearch follows "release window" model as described in this [proposal](https://github.com/opensearch-project/.github/issues/150).  We plan to release a new minor version approximately every six weeks which includes all the new features and fixes that are ready to go. The entry and exit criteria are defined for the artifacts to be released. Every day we generate a release candidate. When the exit criteria are met by a release candidate, we announce, update the release date (~2 days later) and publish release artifacts on that date. 
+For minor version releases, OpenSearch follows "release window" model as described in this [proposal](https://github.com/opensearch-project/.github/issues/150).  The goal is to release a new minor version approximately every six weeks which includes all the new features and fixes that are ready to go. Once we enter a release window, every day we will generate a release candidate. When the exit criteria are met by a release candidate, we will make an announcement, update the release date column above and publish new artifacts on that date. 
 
-**Note:** If we cannot pass the exit criteria by 2 weeks after the start of the release cycle window, we will cancel the minor release and hold changes until the next release window.
+**Note:** If we cannot pass the exit criteria by 2 weeks after the start of the release cycle window (1 week for 1.x releases), we will cancel the minor release and hold changes until the next release window.
```

Comment:
We only do patch release in 1.x line. I assume the updated release process is only valid for minor and patch release will still follow release train model?

Reply:
If I updated it to being "1 week for 1.x patch releases", would that be more or less clear?

Reply:
Yeah, Patch release still follows release train model so just want to make sure that no one gets confused with updated minor release process.

Reply:
okay, updated.  Please review, LMK if there are any other changes.  Thanks!